packages/gatsby/src/redux/machines/page-component.js

```diff
@@ -0,0 +1,143 @@
+const {
+  Machine,
+  actions: { assign },
```

Comment:
```suggestion
  assign,
```
docs write to directly[ import from xstate](https://xstate.js.org/docs/guides/context.html#updating-context-with-assign)

It's the same thing so does not make a big difference
https://github.com/davidkpiano/xstate/blob/master/src/index.ts#L21-L46